generator/uscore/metadata_extractor.rb

```diff
@@ -23,7 +23,7 @@ def extract_metadata
         capability_statement_json = capability_statement('server')
         add_metadata_from_ig(metadata, ig_resource)
         add_metadata_from_resources(metadata, capability_statement_json['rest'][0]['resource'])
-        fix_metadata_errors(metadata)
+        # fix_metadata_errors(metadata)
```

Comment:
If this isn't needed anymore, just remove it completely rather than commenting it out.

Reply:
Yeah I understand why it is nice to have it in here but commented out (an easy place to know where you are supposed to add metadata errors in case they pop up again), but I think we need to trust that we can figure out how to readd this if others show up without having to have this cluttering things up in the meantime.  I vote to remove this comment and the method.